Sources/HexGrid/Convertor.swift

```diff
@@ -77,8 +77,8 @@ internal struct Convertor {
         orientation: Orientation) -> Point {
         let axialCoords = coordinates.toAxial()
         let orientation = OrientationMatrix(orientation: orientation)
-        let x = ((orientation.f00 * Double(axialCoords.q)) + (orientation.f10 * Double(axialCoords.r))) * hexSize.width
-        let y = ((orientation.f01 * Double(axialCoords.q)) + (orientation.f11 * Double(axialCoords.r))) * hexSize.height
+        let x = ((orientation.f00 * Double(axialCoords.q)) + (orientation.f10 * Double(axialCoords.r))) * (hexSize.width/2)
+        let y = ((orientation.f01 * Double(axialCoords.q)) + (orientation.f11 * Double(axialCoords.r))) * (hexSize.height/2)
 
```

Comment:
This was the only functional change

Reply:
Great fix. I am also using this amazing library with SpriteKit and I thought I was using it wrong so I ended up scaling my node radius by 2 to compensate. Just pulled latest and removed my kludge. Thanks to both of you!

Reply:
No worries.  Glad to have contributed!